core/src/main/java/bisq/core/trade/protocol/TradeProtocol.java

```diff
@@ -420,4 +420,14 @@ private void cleanupTradableOnFault() {
             }
         }
     }
+
+    protected boolean wasDisputed(ErrorMessageHandler errorMessageHandler) {
+        if (trade.getDisputeState() != Trade.DisputeState.NO_DISPUTE) {
+            String msg = "Dispute have been opened once. We do not allow anymore to confirm payment by button click.";
+            log.error(msg);
```

Comment:
Error log is too harsh here, info seems more reasonable.

Reply:
I think that is not too harsh as it is not possible to get there as long a user does not write custom code. Will be also more visible in case the API starts to use those code branches and does not do the checks we do in the UI. Maybe even an exception is justified here?

Reply:
But maybe the method does too much. Should only return a boolean and not handle the true case. I will have a look and maybe add a commit to an open PR.